tests/backups/test_manager.py

```diff
@@ -1734,12 +1734,20 @@ async def test_backup_remove_error(
     healthy_expected: bool,
 ):
     """Test removing a backup error."""
-    copy(get_fixture_path("backup_example.tar"), coresys.config.path_backup)
-    await coresys.backups.reload(location=None, filename="backup_example.tar")
+    backup_file = get_fixture_path("backup_example.tar")
+    if location_name is None:
+        copy(backup_file, coresys.config.path_backup)
+        location = None
+    else:
+        (mount_dir := coresys.config.path_mounts / location_name).mkdir()
+        copy(backup_file, mount_dir)
+        location = coresys.mounts.get(location_name)
```

Comment:
Instead of the if-statement, consider adding additional parametrization variable, for the subdirectory and maybe the key to get the location from `coresys.mounts.get`

Reply:
Unfortunately how to get the location is different for None and mount backups. I've reused existing code now, so that makes the if statement unnecessary.